openvdb/math/FiniteDifference.h

```diff
@@ -24,6 +24,12 @@ namespace OPENVDB_VERSION_NAME {
 namespace math {
 
 
+/// @brief  Pi constant taken from Boost to match old behaviour
+template <typename T> inline constexpr T pi() { return 3.141592653589793238462643383279502884e+00; }
+template <> inline constexpr float pi() { return 3.141592653589793238462643383279502884e+00F; }
+template <> inline constexpr double pi() { return 3.141592653589793238462643383279502884e+00; }
+template <> inline constexpr long double pi() { return 3.141592653589793238462643383279502884e+00L; }
```

Comment:
I think this a good idea, I considered proposing this before. I would expect this constant to be defined in math/Math.h though as it's a bit surprising it would be in the FiniteDifference header.

Also, probably worth putting a brief comment for the future to mention that pi will be available as a constant in C++20 (std::numbers::pi).

Reply:
done